source/services/dataContracts/converters/aliasConverter/aliasConveter.tests.ts

```diff
@@ -0,0 +1,20 @@
+import { AliasConverter } from './aliasConverter';
+
+describe('aliasConverter', (): void => {
+	let aliasConverter: AliasConverter<string>;
+
+	beforeEach((): void => {
+		aliasConverter = new AliasConverter<string>('propValue');
+	});
+
+	it('should get the property from the parent context using the specified property name', (): void => {
+		let parent: any = { propValue: 'value' };
+		expect(aliasConverter.fromServer(undefined, parent)).to.equal('value');
+	});
+
+	it('should set the value on the parent context using the specified property name', (): void => {
+		let parent: any = {};
+		aliasConverter.toServer('value', parent);
```

Comment:
Sure, but the issue is that the converterService sets the return value on the property with the same key as the transform.